sorting_algos/selection/selection.py

```diff
@@ -0,0 +1,15 @@
+def selection(lst):
+    """Take unsorted list and return it sorted."""
+    if type(lst) is not list:
+        raise TypeError('argument must be of type <list>')
+
+    for i in range(len(lst) - 1):
+        mini = i
+        tmp = lst[i]
+        for j in range(i + 1, len(lst)):
+            if lst[j] < lst[mini]:
+                mini = j
+        lst[i] = lst[mini]
+        lst[mini] = tmp
```

Comment:
Just so you know, one benefit of Python is that you do not need a `tmp` variable like this in order to do a swap. It can actually be done on one line like so:
```python
lst[i], lst[mini] = lst[mini], lst[i]
```